pom.xml

```diff
@@ -126,5 +119,79 @@
         <hadoop.version>${hadoop3.version}</hadoop.version>
       </properties>
     </profile>
+    <profile>
+      <!-- default HBase profile -->
+      <id>hbase2</id>
+      <activation>
+        <property>
+          <name>!hbase.profile</name>
+        </property>
+      </activation>
+      <properties>
+        <hbase.version>${hbase2.version}</hbase.version>
+      </properties>
+      <!-- For testing against ZK -->
+      <dependencies>
+	<dependency>
+	  <groupId>org.apache.hbase</groupId>
+	  <artifactId>hbase-zookeeper</artifactId>
+	  <version>${hbase2.version}</version>
```

Comment:
There are tabs being used here and spaces everywhere else. I'm fine with either, but let's be consistent.